docs/source/models.rst

```diff
@@ -2528,7 +2528,7 @@ to formats that are compatible with json serialization and casting to Pandas Dat
     types that can be loaded as ``pyfunc``.
 
     In the current version, text-based large language
-    models are supported for use with ``pyfunc``, while computer vision, audio, multi-modal, timeseries,
```

Comment:
Shall we move `audio` next to `text-based`? `large language models for text and audio processing are supported...`

Reply:
ah good catch. Updated!